dsperse/src/run/runner.py

```
@@ -327,36 +285,265 @@ def _generate_run_metadata(self, format: str = "dirs"):
+        cache_lock = threading.Lock()
+        tile_start = time.time()
+        tile_exec_infos = []
+
+        with ThreadPoolExecutor(max_workers=min(32, num_tiles)) as executor:
+            futures = {
+                executor.submit(
+                    self._run_single_tile, i, slice_id, slice_info, tiling, tile_onnx_path, run_dir, tensor_cache, cache_lock
+                ): i for i in range(num_tiles)
+            }
+            for future in as_completed(futures):
+                tile_idx, cache_output_name, ok, output_tensor, t_info = future.result()
```

Ruff (0.14.11)

Unpacked variable `cache_output_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)